keyboards/handwired/dactyl_manuform/4x6/keymaps/scheiklp/keymap.c

```diff
@@ -0,0 +1,59 @@
+/* A K.O,Y layout for the Dactyl Manuform 4x6 Keyboard */ 
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15